docs/apis/user.yaml

```diff
@@ -610,6 +612,372 @@ paths:
+          description: User schema details
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/UserSchema'
+              examples:
+                simple:
+                  summary: Employee schema with simple attributes
+                  value:
+                    id: "660e8400-e29b-41d4-a716-446655440001"
+                    name: "employee"
+                    schema:
+                      firstname:
+                        type: "string"
+                      lastname:
+                        type: "string"
+                      username:
+                        type: "string"
+                        unique: true
+                      password:
+                        type: "string"
+                        regex: "^(?=.*[a-z])(?=.*[A-Z])(?=.*\\d)(?=.*[!@#$%^&*()_+\\-=\\[\\]{};':\"\\\\|,.<>\\/?]).{8,}$"
+                      employeeId:
+                        type: "number"
+                        unique: true
+                      department:
+                        type: "string"
+                        enum: ["Engineering", "Marketing", "Sales", "HR", "Finance", "Operations"]
+                      isActive:
+                        type: "boolean"
+                complex:
+                  summary: Customer schema with complex attributes
+                  value:
+                    id: "550e8400-e29b-41d4-a716-446655440000"
+                    name: "customer"
+                    schema:
+                      email:
+                        type: "string"
+                        unique: true
+                        regex: "^[a-zA-Z0-9._%+-]+@[a-zA-Z0-9.-]+\\.[a-zA-Z]{2,}$"
+                      mobile:
+                        type: "string"
+                        regex: "^\\+?[1-9]\\d{1,14}$"
+                      address:
+                        type: "object"
+                        properties:
+                          street:
+                            type: "string"
+                          city:
+                            type: "string"
+                          zipCode:
+                            type: "string"
+                            regex: "^\\d{5}(-\\d{4})?$"
+                          country:
+                            type: "string"
+                      contactPreferences:
+                        type: "array"
+                        items:
+                          type: "string"
+                          enum: ["email", "sms", "phone", "mail"]
+        "404":
+          description: User schema not found
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+              example:
+                code: "USR-2004"
+                message: "User schema not found"
+                description: "The user schema with the specified ID does not exist"
+        "500":
+          description: Internal server error
+
+    put:
```

Comment:
Put request should have 409 error response as well if we're making the schema name unique